requirements.txt

```diff
@@ -0,0 +1,53 @@
+ply>=3.9
```

Comment:
what's the need for the `requirements.txt` file?

Reply:
@mroutis This is a WIP and the commit is called `debug`, it is just me trying different things to see if it will help or not.

Reply:
Specifically for that one there were a few theories that maybe pip would install that quicker or maybe parallelizing with xars or pipenv would be quicker.